server/src/BGS.Core/BGS.Core.csproj

```diff
@@ -5,6 +5,10 @@
         <RootNamespace>BGS.ApplicationCore</RootNamespace>
     </PropertyGroup>
 
+    <ItemGroup>
+      <PackageReference Include="Autofac" Version="7.1.0" />
```

Comment:
Is it nesessary to use Autofac? What's wrong with original DI?

Reply:
Now - yes. Original DI not supported keyed registrations:
https://weblogs.asp.net/ricardoperes/net-8-dependency-injection-changes-keyed-services
After migrate to NET 8, we can discuss again and remove autofac